backend/src/main.py

```diff
@@ -141,6 +141,53 @@ class UserLoginRequest:
     password: str
 
 
+@dataclass
+class UserRegisterRequest:
+    name: str
+    password: str
+    confirm_password: str
```

Comment:
Cred ca nu-i nevoie de `confirm_password` pe backend; e destul sa faca verificarea aia din frontend.

Reply:
poti sa dai disable la verificarile de pe frontend dar daca voi vreti il pot scoate

Reply:
ai in vedere daca face cineva request-ul manual? gen din postman?

Reply:
sau poti da disable la javascript. sau cum te gandeai la verificari pe frontend?

Reply:
Da, cu javascript sa verifice. Cred ca daca tot facem in react, atunci daca da disable cnv la javascript, nu va merge nimic :)); deci zic sa nu ne punem problema
Iar la requesturi gen din postman, presupunem ca in cazul acela utilizatorul stie ce face

Reply:
Confirmarea e doar ca sa asigure utilizatorul ca si-a scris bine parola. Nu e nevoie sa fie confirmata si pe backend, e de ajuns pe frontend